core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -100,11 +100,13 @@ object ConsumerPerformance {
     var lastReportTime: Long = startMs
     var lastBytesRead = 0L
     var lastMessagesRead = 0L
-    var lastConsumed = System.currentTimeMillis
-    while(messagesRead < count && lastConsumed >= System.currentTimeMillis - timeout) {
+    var lastConsumedTime = System.currentTimeMillis
+
+
+    while(messagesRead < count && System.currentTimeMillis() - lastConsumedTime < timeout) {
```

Comment:
I'm confused what got "fixed" here. The inequality seems the same except you changed it from `<=` to `<` and rearranged it. What exactly is the bug with using the original version with the new consumer?